docs/api.md

```diff
@@ -1032,7 +1032,7 @@ Shortcut for [page.mainFrame().$$(selector)](#frameselector-1).
 
 #### page.$$eval(selector, pageFunction[, ...args])
 - `selector` <[string]> A [selector] to query page for
-- `pageFunction` <[function]> Function to be evaluated in browser context
+- `pageFunction` <[function]\([Array]<[Element]>\)> Function to be evaluated in browser context
```

Comment:
Without it, markdown can think the function is a link. It tends not to because of syntax errors, but I'd rather not depend on it bailing out on its own.